src/Appwrite/Platform/Tasks/Specs.php

```diff
@@ -299,4 +308,53 @@ public function action(string $version, string $mode, Registry $register): void
             }
         }
     }
+
+    private function generateEvents(array $events, $parentModel = null)
```

Comment:
For this logic, is it possible to reuse the existing code in `Events::generateEvents()`?

Reply:
Not sure, I don't know if we have a base patter we can pass into there. I also assumed no because in the old docs website we didn't reuse this logic :/

Reply:
What about re-using logic in here: https://github.com/appwrite/appwrite/blob/main/src/Appwrite/Event/Validator/Event.php#L34

Reply:
Oh this is worth taking a look.